avalon/maya/lib.py

```diff
@@ -428,7 +447,10 @@ def lsattr(attr, value=None):
     """
 
     if value is None:
-        return cmds.ls("*.%s" % attr)
+        return cmds.ls("*.%s" % attr,
```

Comment:
Ensure that when querying without a value the result actually returns the object's node name _without!_ the attribute in it, so it's consistent with how ` lsattrs` works with values.